kong/runloop/handler.lua

```diff
@@ -1513,7 +1513,7 @@ return {
       -- Report HTTP status for health checks
       local balancer_data = ctx.balancer_data
       if balancer_data and balancer_data.balancer_handle then
-        local status = ngx.status
+        local status = ngx.var.upstream_status
```

Comment:
If an upstream retry occurs, then `ngx.var.upstream_status` will be an array containing the upstream status of each retry, is there any special handling required?

Reply:
Yeah, I just check lua-nginx-module code, It's a string that contains upstream stats code array, I will change this way to solve it.